content/blog/2023-12-12-Update-on-the-last-three-months-in-GR/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Update on the Last Three Months in GNU Radio"
+author: "Marcus Müller"
+date: "2023-12-12"
+aliases: ["blog/update-on-2023-September-to-December"]
+---
+
+We tried to establish a bit of a newsletter. Here's what I've written down about what I think a wider audience would want to know *after* GRCon'23
+
+GNU Radio Releases and Code Highlights
+======================================
+
+GNU Radio
+---------
+
+We released 3.10.8.0 and 3.9.8.0; the latter is the last of the quarterly series of 3.9.x releases. Since 3.10 and 3.9 are mostly code-compatible, we feel confident users of 3.9 should upgrade!
```

Comment:
3.9.8.0??? Is there some cut/paste going on here?

Reply:
Indeed! My bad. Need to fix that in the morning 🤦